src/gui/src/lib/external-info.ts

```diff
@@ -393,6 +433,9 @@ export async function* fetchDetails(
 
     // retrieve download range for the last year from the registry
     trackPromise(fetchDownloadsLastYear())
+
+    // retrieve contributors from the manifest
+    trackPromise(fetchContributors())
```

Comment:
I think this needs to be limited to public-registry-only packages, it's better to move it to outside this if block